CHANGES.md

```diff
@@ -1,5 +1,18 @@
 # Changes
 
+## edge-21.4.3
+
+This edge supersedes `edge-21.4.2` as a release candidate for `stable-2.10.1`!
+
+This release adds support for TrafficSplit `v1alpha2`. `v1alpha1` will be
```

Comment:
might be better to avoid saying anything about our future plans since we haven't figured out exactly what we will be doing yet.

Reply:
Updated